File: backoffice/FondaDomain/Domain/RestaurantCategory.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Runtime.Serialization;

namespace com.ds201625.fonda.Domain
{
    /// <summary>
    /// Representa el tipo de comida que prepara el Restaurante
    /// </summary>
    public class RestaurantCategory : NounBaseEntity
    {
        /// <summary>
        /// Nombre de la categoria del restaurant
        /// </summary>
        private string _nameCategory;

        /// <summary>
        /// Constructor.
        /// </summary>
        public RestaurantCategory() : base() { }

        [DataMember]
        public virtual string NameCategory
        {
            /// <summary>
            /// Obtiene el Nombre de la categoria del restaurante
            /// </summary>
            get { return _nameCategory; }
            /// <summary>
            /// Asigna el Nombre de la categoria del restaurante
            /// </summary>
            /// <value>Recibe el Nombre de la categoria del restaurante</value>
            set { _nameCategory = value; }
        }

    }
}

File: frontoffice/app/src/main/java/com/ds201625/fonda/domains/entities/RestaurantCategory.java
package com.ds201625.fonda.domains.entities;

/**
 * Created by Adri on 06/05/2016.
 */
public class RestaurantCategory extends NounBaseEntity {

    private String nameCategory;

    public RestaurantCategory () {
        super();
    }

    public RestaurantCategory (String nameCategory) {
        this.nameCategory = nameCategory;
    }

    public String getNameCategory() {
        return nameCategory;
    }

    public void setNameCategory(String nameCategory) {
        this.nameCategory = nameCategory;
    }

}
